contrib/builder/rpm/amd64/photon-1.0RC/Dockerfile

```diff
@@ -0,0 +1,18 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/amd64/generate.sh"!
+#
+
+FROM photon:1.0RC
```

Comment:
How long will `1.0RC` be supported?  What impact will this have on the release script (https://github.com/docker/docker/blob/master/hack/make/release-rpm) and artifacts after 1.0 becomes GA, etc?